backend/src/db/migrations/20230828103237_create_genres_table.ts

```diff
@@ -0,0 +1,58 @@
+import { type Knex } from 'knex';
+
+const GENRES = [
+  { name: 'Science Fiction', key: 'sci_fi' },
+  { name: 'Mystery', key: 'mystery' },
+  { name: 'Fantasy', key: 'fantasy' },
+  { name: 'Romance', key: 'romance' },
+  { name: 'Thriller', key: 'thriller' },
+  { name: 'Historical Fiction', key: 'hist_fiction' },
+  { name: 'Horror', key: 'horror' },
+  { name: 'Adventure', key: 'adventure' },
+  { name: 'Biography', key: 'biography' },
+  { name: 'Comedy', key: 'comedy' },
+  { name: 'Dystopian', key: 'dystopian' },
+  { name: 'Non-fiction', key: 'non_fiction' },
+  { name: 'Paranormal', key: 'paranormal' },
+  { name: 'Satire', key: 'satire' },
+  { name: 'Young Adult', key: 'ya_fiction' },
+  { name: 'Crime', key: 'crime' },
+  { name: 'Drama', key: 'drama' },
+  { name: 'Science', key: 'science' },
+  { name: 'Urban Fantasy', key: 'urban_fantasy' },
+  { name: 'Memoir', key: 'memoir' },
+];
```

Comment:
Should not it be moved to seeds?

Reply:
Nope, seeds are not the best choice for deployed app data, as it should be consistent